x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/resource_findings_container.tsx

```diff
@@ -77,9 +83,14 @@ export const ResourceFindings = ({ dataView }: { dataView: DataView }) => {
         </PageTitle>
         <EuiSpacer />
         <ResourceFindingsTable
-          loading={resourceFindings.isLoading}
+          size={urlQuery.size}
+          from={urlQuery.from}
+          loading={resourceFindings.isFetching}
           data={resourceFindings.data}
           error={resourceFindings.error}
+          setPagination={(pagination) =>
+            setUrlQuery({ from: pagination?.index, size: pagination?.size })
```

Comment:
from should be calculated `(index - 1) * size`

Reply:
fixed, shared between `group-by-none` and `resource-findings` tables